gadget/install/kernel.go

```diff
@@ -49,3 +51,52 @@ type KernelModulesComponentInfo struct {
 	// MountPoint is the root of the files from the component
 	MountPoint string
 }
+
+// CompSeedInfo contains information for a component from the seed and
+// from its metadata.
+type CompSeedInfo struct {
+	CompInfo *snap.ComponentInfo
+	CompSeed *seed.Component
+}
+
+// KernelBootInfo contains information related to the kernel used on installation.
+type KernelBootInfo struct {
+	KSnapInfo     *KernelSnapInfo
+	BootableKMods []boot.BootableKModsComponents
+}
+
+// BuildKernelBootInfo constructs a KernelBootInfo.
+func BuildKernelBootInfo(kernInfo *snap.Info, compSeedInfos []CompSeedInfo, kernMntPoint string, mntPtForComps map[string]string, isCore, needsDriversTree bool) KernelBootInfo {
```

Comment:
these are not used in gadget/install, maybe they should go one level up into overlord/install ?

Reply:
Moved